client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -844,34 +851,72 @@ export class PlanFeatures2023Grid extends Component<
 			} );
 	}
 
-	renderPlanStorageOptions( planPropertiesObj: PlanProperties[], options?: PlanRowOptions ) {
+	renderStorageAddOnDropdown( properties: PlanProperties ) {
```

Comment:
I suggest we turn this into a function component `StorageAddOnDropdown`. It's something that will happen in a follow-up, so best just do it now while at it. :-)

Reply:
> I suggest we turn this into a function component StorageAddOnDropdown

Sure -- I don't mind since it's quick and easy, but to make sure I understand, is it because we're trying to migrate away from a _class based_ plans features 2023 grid?

Reply:
Yes, hopefully soon.